src/main/scala/org/ergoplatform/http/api/MiningApiRoute.scala

```diff
@@ -63,18 +64,19 @@ case class MiningApiRoute(miner: ActorRef,
   }
 
   def rewardAddressR: Route = (path("rewardAddress") & get) {
-    val addressF = (miner ? ErgoMiner.ReadMinerPk).mapTo[Option[ProveDlog]]
-      .flatMap {
-        _.fold[Future[ErgoAddress]](Future.failed(new Exception("Miner PK not initialized")))(pk => {
```

Comment:
@kushti Ok, I understand your concern, it used to throw error and now it times out ... do you want me to keep it backward compatible?